### PR TITLE
docs: update component READMEs

### PR DESCRIPTION
**Pull request by benelan**
This PR was automatically generated by the update-doc GitHub action

### CODE REVIEW

File: src/components/calcite-input-message/readme.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ Displays a contextual message to a user. Allows the passing of content, links, e
 ### Basic
 
 ```html
-<calcite-label>
+<calcite-label status="“invalid”">
   My great label
-  <calcite-input placeholder="“Enter" your information” status="invalid"></calcite-input>
-  <calcite-input-message active status="invalid"
+  <calcite-input placeholder="“Enter" your information”></calcite-input>
```
Comment (benelan) [inline src/components/calcite-input-message/readme.md:14]: Is this a typo?

Comment (jcfranco) [reply src/components/calcite-input-message/readme.md:14]: Oh, looks like it. 😱 Sneaky fancy quote!

Comment (driskull) [reply src/components/calcite-input-message/readme.md:14]: lol. This example...

> “Enter" your information”
> That's not going to work out.

:)

Comment (benelan) [reply src/components/calcite-input-message/readme.md:14]: well you're not wrong... it is invalid after all

**Comment by benelan** [comment]
closing this since `status` is deprecated on `calcite-label` via #3984

**Pull request closed without merging**